.github/workflows/interface-unit-tests.yml

```
@@ -232,12 +281,13 @@ jobs:
       install_pennylane_lightning_master: true
       pytest_coverage_flags: ${{ inputs.pytest_coverage_flags }}
       pytest_markers: tf and not qcut and not finite-diff and not param-shift
-      pytest_additional_args: --splits 3 --group ${{ matrix.group }}
-      pytest_durations_file_path: '.github/workflows/tf_tests_durations.json'
+      pytest_additional_args: ${{ needs.setup-ci-load.outputs.enable_split == '1' && format('--splits {0} --group {1}', '3', matrix.group) || '' }}
+      pytest_durations_file_path: ${{ needs.setup-ci-load.outputs.enable_split == '1' && '.github/workflows/tf_tests_durations.json' || '' }}
```

```suggestion
      pytest_durations_file_path: '.github/workflows/tf_tests_durations.json'
```

The update to the durations file path isn't needed. We only save the durations artifacts if the `pytest_store_durations` input is `true` (`false` by default), and since we're not splitting, specifying the durations file wouldn't do anything regardless.

---

So safe to keep in without impacting anything --- all good then.